content/tab-programming.html.bak

```diff
@@ -0,0 +1,1073 @@
+<div id="tab-programming-container">
```

Comment:
Tämän varmuuskopiotiedoston voisi poistaa.

Reply:
Tähän osaa vastata käyttäjä @mplattu

Reply:
Tiedosto content/tab-programming.html.bak näyttäisi syntyneen vahingossa samalla, kun on 
lakattu puhumasta argumenteista commitissa 45462cbfbf7aa32079fbf87891a5d871ad05fdbf ("Only speak of parameters").